rest/route/github_routes.go

```diff
@@ -101,22 +101,26 @@ func (gh *githubHookApi) Execute(ctx context.Context, sc data.Connector) (Respon
 				Message:    "pull request has no action",
 			}
 		}
+
 		if *event.Action == "opened" || *event.Action == "synchronize" ||
 			*event.Action == "reopened" {
 			ghi, err := patch.NewGithubIntent(gh.msgId, event)
 			if err != nil {
-				return ResponseData{}, rest.APIError{
+				return ResponseData{}, &rest.APIError{
 					StatusCode: http.StatusBadRequest,
 					Message:    err.Error(),
 				}
 			}
 
 			if err := sc.AddPatchIntent(ghi, gh.queue); err != nil {
-				return ResponseData{}, rest.APIError{
+				return ResponseData{}, &rest.APIError{
 					StatusCode: http.StatusInternalServerError,
 					Message:    err.Error(),
 				}
 			}
+
+		} else if *event.Action == "closed" {
```

Comment:
is closed a constant somewhere?

Reply:
go-github doesn't offer any constants for these. I'll add constants for these.